### ISSUE TITLE
Group all contributions to the overall relative transmissivity

### ISSUE THREAD

**Issue by SaraRegibo** [IvS-KULeuven/PlatoSim3#478]
The contribution to the relative transmissivity (i.e. depending on the distance from the OA) is three-fold:

1.  natural vignetting (due to view factor);
2. mechanical vignetting (due to undersized mask at the entrance window);
3. glass absorption + AR coating.

We have implemented (1) and part of (2) as "natural vignetting" and "mechanical vignetting" resp., but we should really implement one single model for the relative transmissivity that incorporates all three components.  The model proposed by Carsten is a polynomial fit:

> f(x) = 4.18e-2 * x^2 – 5.65e-5 * x^4 + 2.37e-7 * x^6

This means that we would have to throw out the whole `Vignetting` block in the configuration file and replace it with a new block comprising the information on the polynomial model.

**Comment by SaraRegibo** [comment]
Update to the configuration files:
* Removal of all parameters regarding (natural/mechanical) vignetting
* Add block for overall relative transmissivity (+ switch):
```
RelativeTransmissivity:
        Coefficients:           [4.18e-2, -5.65e-5, 2.37e-7]    # Coefficients of the polynomial fit to the variation in overall relative transmissivity
        RadiusFOV:              18.8908                         # Radius of the FOV [degrees]
        ExpectedValue:          0.920                           # Expected value of the efficiency (mean of all detector pixels within FOV)
IncludeRelativeTransmissivity:   yes             # Include the overall relative transmissitiy (natural + mechanical vignetting, glass absorption, and anti-reflective coating)
```

**Comment by SaraRegibo** [comment]
As this is related to issue #470, assigning to @s-geiser for checking.  Please, have a look at pull request #479.

**Comment by s-geiser** [comment]
Looks fine to me - Could you add a comment in the inputfiles mentioning which polynomials are provided (x^2/x^4/x^6) and mention that it is radial change that is covered? I think otherwise it will be difficult to understand later on.

**Comment by SaraRegibo** [comment]
Thanks for checking!  I've extended the comment in all configuration files.

**Comment by SaraRegibo** [comment]
Closed by #479.
